### DISCUSSION TITLE
为什么drivers里lark驱动没有在all.go里

### DISCUSSION THREAD

**Discussion by Sakana-1314** [AlistGo/alist#7225] [General]
为什么drivers里lark驱动没有在all.go里，而是单独的lark.go，它是有什么不同吗

**Comment by anwen-anyi** [comment]
只有`x64` 和 `arm64`的版本AList有这个驱动，别的没有，详情看下面
 - https://github.com/alist-org/alist/pull/6475#issuecomment-2119560240
